onlineweb4/settings/base.py

```diff
@@ -115,7 +115,7 @@
 
 # Django CORS headers
 CORS_ORIGIN_ALLOW_ALL = config("OW4_DJANGO_CORS_ORIGIN_ALLOW_ALL", cast=bool, default=True)
-CORS_URLS_REGEX = r'^(/api/v1/.*|/sso/user/)$' # Enables CORS on /api/v1/ endpoints and the /sso/user/ endpoint
+CORS_URLS_REGEX = r'^(/api/v1/.*|/sso/user/|/openid/.*)$' # Enables CORS on /api/v1/ endpoints and the /sso/user/ endpoint
```

Comment:
I'm gonna whine again and ask you to update the comment too, so it matches the code.